doc/release-notes/0.20/0.20.md

```diff
@@ -0,0 +1,114 @@
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 version 0.20.0 release notes
+
+These release notes support the [Eclipse OpenJ9 0.20.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.20.0/plan).
+
+## Binaries and supported environments
+
+OpenJ9 release 0.20.0 supports OpenJDK 8, 11, and 14.
+
+Binaries are available at the AdoptOpenJDK project:
+
+- [OpenJDK 8 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk8&jvmVariant=openj9)
+- [OpenJDK 11 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk11&jvmVariant=openj9)
+- [OpenJDK 14 with OpenJ9](https://adoptopenjdk.net/archive.html?variant=openjdk14&jvmVariant=openj9)
+
+All builds are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests at AdoptOpenJDK.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.20.0. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.20/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/8498">#8498</a></td>
+<td valign="top">Change to default maximum heap size for Java 8</td>
+<td valign="top">OpenJDK8 only</td>
+<td valign="top">For consistency with Java&trade; 11, the default maximum heap size (<tt>-Xmx</tt>) is changed to be 25% of the available memory with a maximum of 25 GB. Where there is 2 GB or less of physical memory, the value set is 50% of available memory with a minimum value of 16 MB and a maximum value of 512 MB. If you want to revert to the default setting in earlier releases of OpenJ9, use the <tt>-XX:+OriginalJDK8HeapSizeCompatibilityMode</tt> option.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/8886">#8886</a></td>
+<td valign="top">Change to default maximum heap size for Java 8</td>
```

Comment:
This description doesn't match the impact.

Reply:
I took this from the doc change, so sounds like we need more work there too. I dug deeper into the originating issue and added some impacts that Dmitri identified. It sounds a little open ended though. "If my app is impacted, what tuning might I do to avoid using the -XX option to revert". Any suggestions welcome.

Reply:
I think you misunderstood my comment, but the mismatching description and impact for ExitOnOutOfMemoryError has been fixed.